pa11y/yarn.lock

```diff
@@ -0,0 +1,822 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove this file if it is no longer relevant.

As far as I can tell is it made irrelevant by using the root `package.json`.

Reply:
Done